src/package_io/parse_input.star

```diff
@@ -45,6 +45,7 @@ def parse_input(input_args):
 				participants.append(new_participant)
 			result["participants"] = participants
 
+	total_participant_count = 1
```

Comment:
@adschwartz shouldn't this start at 0 rather than 1?

Reply:
Yeah the default needs to be `1` but only if there no `count`s defined. The way I did it was wrong so fixing that. Thank you!